presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveArrayFunctionQueries.java

```diff
@@ -37,4 +37,12 @@ public void testRepeat()
         this.assertQueryFails("SELECT repeat(orderkey, 10001) FROM lineitem",
                 ".*Count argument of repeat function must be less than or equal to 10000.*");
     }
+
+    @Test
+    public void testShuffle()
+    {
+        this.assertQuery("SELECT shuffle(quantities), shuffle(quantity_by_linenumber) FROM orders_ex");
```

Comment:
Wouldn't this query fail since `quantity_by_linenumber` is a map instead of `array`?

Reply:
Thanks for reviewing my PR, @zacw7 !
Good question. I'm unfamiliar with the tpch table schemas, where can I find those schema definition?

Reply:
You can find the definition of `orders_ex` in [here](https://github.com/prestodb/presto/blob/818df29d75cbd67d47a67178f18afaaea838be18/presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java#L233-L235).